setuptools/command/easy_install.py

```diff
@@ -1175,23 +1174,8 @@ def report_editable(self, spec, setup_script):
         python = sys.executable
         return '\n' + self.__editable_msg % locals()
 
-    def run_setup(self, setup_script, setup_base, args) -> None:
-        sys.modules.setdefault('distutils.command.bdist_egg', bdist_egg)
-        sys.modules.setdefault('distutils.command.egg_info', egg_info)
-
-        args = list(args)
-        if self.verbose > 2:
-            v = 'v' * (self.verbose - 1)
-            args.insert(0, '-' + v)
-        elif self.verbose < 2:
-            args.insert(0, '-q')
-        if self.dry_run:
-            args.insert(0, '-n')
-        log.info("Running %s %s", setup_script[len(setup_base) + 1 :], ' '.join(args))
-        try:
-            run_setup(setup_script, args)
-        except SystemExit as v:
-            raise DistutilsError(f"Setup script exited with {v.args[0]}") from v
+    def run_setup(self, setup_script, setup_base, args):
```

Comment:
If you're wondering about the return annotation here to replace the old `-> None, it would be:

```suggestion
    def run_setup(self, setup_script, setup_base, args) -> NoReturn:
```